packages/jsii-calc/lib/compliance.ts

```diff
@@ -240,6 +240,10 @@ export class RuntimeTypeChecking {
     public methodWithOptionalArguments(arg1: number, arg2: string, arg3?: Date) {
         arg1; arg2; arg3;
     }
+
+    public methodWithDefaultedArguments(arg1: number = 2, arg2: string, arg3: Date = new Date()) {
```

Comment:
Can you add a test to verify this for ctors as well?

Reply:
Done!